docs/source/getting_started/installation.md

```diff
@@ -5,12 +5,52 @@ installation of the plugin and its dependencies. `uv` provides advanced
 dependency resolution which is required to properly install dependencies like
 `vllm` without overwriting critical dependencies like `torch`.
 
-```bash
-# Install uv
-pip install uv
-
-# Install vllm-spyre
-git clone https://github.com/vllm-project/vllm-spyre.git
-cd vllm-spyre
-VLLM_TARGET_DEVICE=empty uv pip install -e .
-```
+1. Clone vllm-spyre
+
+   ```sh
+   git clone https://github.com/vllm-project/vllm-spyre.git
+   cd vllm-spyre
+   ```
+
+1. Install uv
+  
+   ```sh
+   pip install uv
+   ```
+  
+1. Create a new env
+
+   ```sh
+   uv venv --python 3.12 --seed .venv
+   ```
+
+1. Activate it
+  
+   ```sh
+   source .venv/bin/activate
+   ```
+
+1. Install `vllm-spyre` locally with dev (and optionally lint) dependencies
+  
+   ```sh
+   uv sync --frozen --active
```

Comment:
`VLLM_TARGET_DEVICE=empty` needs to be set before building the project, right?

Reply:
I have never set that variable but it still works for me though.

Reply:
Can you try it out?